addons/test_mail/tests/test_mail_activity.py

```diff
@@ -781,6 +781,20 @@ def test_record_unlink(self):
         test_record.unlink()
         self.assertFalse((act1 + act2).exists(), 'Removing records should remove activities, even archived')
 
+    @users('employee')
+    def test_fix_orphan_activities(self):
```

Comment:
Beware alphabetical ordering :D !
```suggestion
    def test_record_unlinked_orphan_activities(self):
```